lib/fetcher.js

```diff
@@ -61,7 +61,8 @@ class FetcherBase {
     // by adding/modifying the integrity value.
     this.opts = { ...opts }
 
-    this.cache = opts.cache || cacheDir()
+    this.cache = opts.cache || cacheDir().cacache
```

Comment:
Why are we calling this twice?

Reply:
Ok these are passed by default from npm so this is already optimized for that use case calling a tiny function twice outside that use case is ok